prime-dotnet-webapi/HttpClients/Org Book API/OrgBookClient.cs

```diff
@@ -0,0 +1,66 @@
+using System;
+using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+using System.Collections.Generic;
+using System.Security.Cryptography.X509Certificates;
+
+using Prime.HttpClients.PharmanetCollegeApiDefinitions;
+using System.Web;
+using Newtonsoft.Json.Linq;
+using Newtonsoft.Json;
```

Comment:
VS Code is indicating some of these aren't used

Reply:
Unused references are removed.